Adding GMP and Cloud Monitoring for gke-batch-refarch (#856)
GoogleCloudPlatform/ai-on-gke: best-practices/gke-batch-refarch/02_platform/monitoring/gmp/README.md

# Monitoring Kueue with Google Managed Prometheus and Cloud Monitoring

This document describes how to monitor Kueue metrics using Google Managed Prometheus and Cloud Monitoring.

## Overview

> Kueue metrics are automatically collected by Google Managed Prometheus using managed collection. This means Prometheus-based collectors run as a Daemonset, ensuring scalability by only scraping targets on colocated nodes. The collected metrics are then exported and made available in Google Cloud's Monitoring service.

Reviewer: ```suggestion
You can configure Google Managed Prometheus to automatically collect Kueue metrics. The collected metrics are then exported and made available in Google Cloud's Monitoring service.
```

The rest seems to be documentation for GMP itself, which is not relevant here.

## Viewing the Dashboard

The Kueue dashboard is available in Google Cloud Monitoring. This dashboard provides a visual representation of key Kueue metrics, allowing you to quickly assess the health and performance of your Kueue system.

<img src="../../../images/kueue_cloud_monitoring_1.png" width="800">
<img src="../../../images/kueue_cloud_monitoring_2.png" width="800">

## Configuring Managed Collection and Creating the Dashboard

> The following command will configure Managed Collection for Kueue and Create the Dashboard in Cloud Monitoring. 

Reviewer: ```suggestion
Run the following command to configure Managed Collection for Kueue and Create the Dashboard in Cloud Monitoring. 
```

Also, shouldn't this section be before "Viewing the Dashaboard"?
